changes/9195.resample.rst

```diff
@@ -0,0 +1 @@
+Clarified documentation to explicitly state that ``crval`` values to the ``resample`` step are 0-based image coordinates.
```

Comment:
it might be ok to just remove this and use the `no-changelog-entry-needed` tag instead